src/features/parameters/dynamic-simulation/curve/common/checkbox-treeview.tsx

```
@@ -6,11 +6,11 @@
  */
 
 import React, { forwardRef, Ref, useCallback, useEffect, useImperativeHandle, useMemo, useState } from 'react';
-import { alpha, Checkbox, styled, SxProps, Theme, useTheme } from '@mui/material';
+import { alpha, Checkbox, styled, SxProps, Theme } from '@mui/material';
 import { SimpleTreeView, TreeItem, treeItemClasses } from '@mui/x-tree-view';
 
-const BorderedTreeItem = styled(TreeItem)(({ theme, root }: { theme: Theme; root: boolean }) => {
-    const border = `1px dashed ${alpha(theme.palette.text.primary, 0.4)}`;
+const BorderedTreeItem = styled(TreeItem)(({ theme, root }: { theme?: Theme; root: boolean }) => {
+    const border = `1px dashed ${alpha(theme?.palette.text.primary!, 0.4)}`;
```

Biome (2.5.0)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)